src/environments/environment.staging.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
   production: true,
   apiUrl: 'https://xby2-website-api-staging.azurewebsites.net',
-  googleAnalyticsTrackingId: 'UA-114675726-1'
+  googleAnalyticsTrackingId: 'UA-114675726-1',
+  baseCmsUrl: 'http://localhost/xby2/wp-json/xby2/v1'
```

Comment:
Can this be changed to use the following:

'https://xby2-website-cms-staging.azurewebsites.net/wp-json/xby2/v1'